Lib.Web.Mvc/JQuery/JqGrid/JqGridBootstrap4IconSet.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Lib.Web.Mvc.JQuery.JqGrid
+{
+    /// <summary>
+    /// Icon set for Bootstrap 4 style
+    /// </summary>
+    public enum JqGridBootstrap4IconSet
+    {
+        /// <summary>
+        /// Use Iconic pack
+        /// </summary>
+        Iconic,
+        /// <summary>
+        /// Use Octicons pack
+        /// </summary>
+        Octicons,
+        /// <summary>
+        /// Use FontAwesome pack
+        /// </summary>
+        /// <remarks>Available since Gurido JqGrid 5.3.1</remarks>
+        fontAwesome
```

Comment:
This one breaks naming convetion, enum values should use  Pascal case (I understand it will require dedicated handling in serialization and rendering).